corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -53,16 +53,9 @@ def __call__(self, user, version, last_sync=None, app=None):
         root.append(reports_elem)
         return [root]
 
-    @staticmethod
-    def _report_config_to_fixture(report_config, user):
-        report_elem = E.report(id=report_config.uuid)
-        try:
-            report = get_report_config(report_config.report_id, user.domain)[0]
-        except ResourceNotFound as err:
-            # ReportConfiguration not found
-            raise BadReportConfigurationError('Error getting ReportConfiguration with ID "{}": {}'.format(
-                report_config.report_id, err))
-        data_source = ReportFactory.from_spec(report)
+    @classmethod
```

Comment:
it looks like everything here can still be made to use `@staticmethod`. any reason you changed it?